app/dashboard/src/providers/ProjectsProvider.tsx

```diff
@@ -114,6 +114,7 @@ export default function ProjectsProvider(props: ProjectsProviderProps) {
 function PageSynchronizer() {
```

Comment:
I know it's unrelated but why do we need to synchronize the state? 
Most of the problems are either in async code or synchronizing logic.

Reply:
umm... yeah actually idk if this is needed anymore. iirc originally `page` needed `launchedProjects` for validation but i think this is is no longer the case. (so it should be fine to remove it from the store completely)

that said, idk if it makes sense to do here, perhaps we should add it as a new issue?

Reply:
Yes, it's unrelated to the PR. we can do this in another PR